Src/WitsmlExplorer.Frontend/services/apiClient.tsx

```diff
@@ -38,11 +37,12 @@ export class ApiClient {
     pathName: string,
     abortSignal: AbortSignal | null = null,
     currentCredentials = CredentialsService.getCredentials(),
-    includeCredentials = false
+    includeCredentials = true,
```

Comment:
includeCredentials is AFAIK always set to true and can thus be remove from the signature

Reply:
true, i think also the `authorization`endpoint uses it now. I will remove it thanks!